libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -687,6 +738,13 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           },
           lifecycle: pruneAfterDays(970),
           progress: 1,
+          onExit: defineTemplateApi({
+            action:
+              ApiModuleActions.validateApplication &&
+              ApiModuleActions.setBirthDate,
```

Comment:
I don't think we will get the birth date when vmst approves the application.

Reply:
No, but we check for it when we are in the approved state, and click the link to residence grant